app.R

```diff
@@ -0,0 +1,843 @@
+# Load Packages ----
+library(shiny)
+library(shinydashboard)
+library(shinyBS)
+library(shinyWidgets)
+library(boastUtils)
+library(ggplot2)
+library(shinycssloaders)
+library(dplyr)
+
+# Global Constants, Functions, and Data ----
+fixedColors <- c(
+  "UK-Males" = psuPalette[1],
+  "UK-Females" = psuPalette[2],
+  "US-Males" = psuPalette[3],
+  "US-Females" = psuPalette[4],
+  "China-Males" = psuPalette[6],
+  "China-Females" = psuPalette[7]
+)
+
+countryColors <- c(
+  "United Kingdom" = psuPalette[2],
+  "United States" = psuPalette[4],
+  "China" = psuPalette[7]
+)
+
+load(file = "allData.RData")
+
+
+# Define UI for App ----
+ui <- list(
+  dashboardPage(
+    skin = "green",
+    ### Header ----
+    dashboardHeader(
+      title = "Life Tables",
+      titleWidth = 250,
+      tags$li(class = "dropdown", actionLink("info", icon("info"))),
+      tags$li(
+        class = "dropdown",
+        boastUtils::surveyLink(name = "Life_Table")
+      ),
+      tags$li(
+        class = "dropdown",
+        tags$a(href = "https://shinyapps.science.psu.edu/",
+               icon("home")
+        )
+      )
+    ),
+    ### Sidebar ----
+    dashboardSidebar(
+      width = 250,
+      sidebarMenu(
+        id = "pages",
+        menuItem("Overview", tabName = "overview",icon = icon("dashboard")),
+        menuItem("Survival Rate", tabName = "survivalRate", icon = icon("wpexplorer")),
+        menuItem("Static Pop. Pyramids", tabName = "staticPry", icon = icon("wpexplorer")),
+        menuItem("Time & Pop. Pyramids", tabName = "timePry", icon = icon("wpexplorer")),
+        menuItem("Fecundity Rate", tabName = "fecundityRate", icon = icon("wpexplorer")),
+        menuItem("References", tabName = "references", icon = icon("leanpub"))
+      ),
+      tags$div(
+        class = "sidebar-logo",
+        boastUtils::sidebarFooter()
+      )
+    ),
+    ### Body ----
+    dashboardBody(
+      tabItems(
+        #### Overview Page ----
+        tabItem(
+          tabName = "overview",
+          h1("Life Tables"),
+          p("In this app, you will explore various ways to visualize life tables
+            for 3 different countries--China, the United Kingdom, and the United
+            States of America. You'll be able to look at the several different
+            metrics associated with life tables (e.g., survival and fecundity
+            rates) by country and by sex."),
+          h2("Instructions"),
+          p("Use the left sidebar menu to explore the following life tables."),
+          tags$ul(
+            tags$li(tags$strong("Survival Rate:"), " Click the country and sex
+                    combination you preferred to compare, and view the distinct
```

Comment:
"you prefer", everything else seems to be in present tense

Reply:
Fixed.